biome.jsonc

```diff
@@ -25,26 +25,15 @@
     "rules": {
       "recommended": true,
       "complexity": {
-        "noForEach": "off",
-        "noStaticOnlyClass": "off",
-        "noThisInStatic": "off",
+        // We need couple of empty exports to make those files as esm modules
         "noUselessEmptyExport": "off",
-        "noUselessTypeConstraint": "error",
+        // We use normal functions for to ease with debugging context
```

Comment:
we use named functions to ease with debugging, but that should be covered by the rule,  "Named function expressions are ignored" https://biomejs.dev/linter/rules/use-arrow-function/#_top

Reply:
Done